app/main.py

```diff
@@ -1,2 +1,116 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Any, Hashable, Iterable
+import math
```

Comment:
I bet you don't need the whole math module, do you? You can import only `floor` if you really need it..